### PR TITLE
feat: add basic support openApi 3.1

### PR DESCRIPTION
**Pull request by AlexVarchuk**
## What/Why/How?
Add basic support to openApi 3.1. 
fixed https://github.com/Redocly/openapi-cli/issues/276
fixed https://github.com/Redocly/redoc/issues/1327

## Reference
Basic OpenApi 3.1 support includes:
 - preventing crashes in version 3.1
 - type can be an array of strings
 - add null type
 - add webhooks supporting
 - add summary and identifier to license
 - add resolve `pathItems` for webhooks
 - add support for the type of the number in exclusiveMinimum/Maximum fields
 - add const keyword
 
## Check yourself

- [x] Code is linted
- [x] Tested
- [x] Add tests

### CODE REVIEW

**Comment by AntonKozachuk** [comment]
Fixes #1503

File: demo/index.tsx [hunk 1]
```diff
@@ -5,7 +5,11 @@ import { resolve as urlResolve } from 'url';
 import { RedocStandalone } from '../src';
 import ComboBox from './ComboBox';
 
+const DEFAULT_SPEC = 'openapi.yaml';
+const NEW_VERSION_SPEC = 'openapi-3-1.yaml';
+
 const demos = [
+  { value: NEW_VERSION_SPEC, label: 'OpenApi 3.1' },
```
Comment (RomanHotsiy) [inline demo/index.tsx:12]:
```suggestion
  { value: NEW_VERSION_SPEC, label: 'Petstore OpenAPI 3.1' },
```

File: src/components/ApiInfo/ApiInfo.tsx [hunk 2]
```diff
@@ -100,7 +100,8 @@ export class ApiInfo extends React.Component<ApiInfoProps> {
               )) ||
                 null}
             </StyledMarkdownBlock>
-            <Markdown source={store.spec.info.description} data-role="redoc-description" />
+            <Markdown source={store.spec.info.description} data-role="redoc-description"/>
+            <Markdown source={store.spec.info.summary} data-role="redoc-summary"/>
```
Comment (RomanHotsiy) [inline src/components/ApiInfo/ApiInfo.tsx:104]:
```suggestion
            <Markdown source={store.spec.info.summary} data-role="redoc-summary"/>
            <Markdown source={store.spec.info.description} data-role="redoc-description"/>
```

File: src/services/models/Schema.ts [hunk 3]
```diff
@@ -104,22 +105,28 @@ export class SchemaModel {
...
-    this.displayType = this.type;
     this.displayFormat = this.format;
     this.isPrimitive = isPrimitiveType(schema, this.type);
     this.default = schema.default;
     this.readOnly = !!schema.readOnly;
     this.writeOnly = !!schema.writeOnly;
+    this.const = schema.const || '';
+
+    if (!!schema.nullable) {
+      if (Array.isArray(this.type)) this.type.push('null');
```
Comment (RomanHotsiy) [inline src/services/models/Schema.ts:125]:
```suggestion
      if (Array.isArray(this.type) && !this.type.contains('null')) {
        this.type = [...this.type, 'null'];
      }
```

**Review by RomanHotsiy — APPROVED** [review]
Remove the extra console log and it's good to go 👍

**Pull request merged**